payments.md

```diff
@@ -71,8 +71,8 @@ type ReconcileResponse struct {
 
 ## TODO: what are the possible status cases?
 type Status enum {
```

Comment:
unsure what the purpose of this is, by default enums are just strings but we are discussing getting an int enum in but it'll need ints next to each item